src/net.h

```diff
@@ -638,6 +640,9 @@ class CNode
     const uint64_t nKeyedNetGroup;
     std::atomic_bool fPauseRecv;
     std::atomic_bool fPauseSend;
+
+    const NodeId id;
```

Comment:
Is it used somewhere?

Reply:
it is required in derived class as much as I remember